pkg/chartutil/save.go

```diff
@@ -93,7 +93,7 @@ func Save(c *chart.Chart, outDir string) (string, error) {
 	filename := fmt.Sprintf("%s-%s.tgz", c.Name(), c.Metadata.Version)
 	filename = filepath.Join(outDir, filename)
 	if stat, err := os.Stat(filepath.Dir(filename)); os.IsNotExist(err) {
-		if err := os.MkdirAll(filepath.Dir(filename), 0755); !os.IsExist(err) {
+		if err := os.MkdirAll(filepath.Dir(filename), 0755); err != nil {
```

Comment:
Will this fail if the directory already exists? Is this expected  behaviour?

If not then maybe the check should be something like this:
`if err != nil && !os.IsExist(err)`

Reply:
Thanks for reviewing this. It seems that os.MkdirAll will only be called when the outer if fails, i.e. os.IsNotExist == true. So it wouldn't try to mkdir an existing directory, isn't it?

Reply:
That is correct. From API: "If path is already a directory, MkdirAll does nothing and returns nil."